### PR TITLE
[#73] [Android] [UI] As a user, I can see the thanks for taking the survey screen

### LINKED ISSUE #73: [Android] [UI] As a user, I can see the thanks for taking the survey screen
## Why

In the final step, after users submit their responses. The mobile app must show the Thanks screen if the submit response request is successful.

## Acceptance Criteria

- Show the animation on the icon by using Lottie.
- Show the title "Thanks for taking  the survey."

## Design

Image

[Thanks for taking the survey](https://www.figma.com/file/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=36%3A28)

### PR DESCRIPTION
**Pull request by suho**
- Closes #73

## What happened 👀

Created the thanks for taking survey screen with animation and text.

## Insight 📝

- Added Lottie for jetpack compose.

## Proof Of Work 📹

https://user-images.githubusercontent.com/19943832/208608821-497639c8-0834-43c6-83f2-756d60436425.mov

### CODE REVIEW

**Review by hoangnguyen92dn — APPROVED** [review]
The rest LGTM 🚀

File: android/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -14,4 +14,5 @@
     <string name="submit_button">Submit</string>
     <string name="nps_not_at_all_likely">Not at all Likely</string>
     <string name="nps_extremely_likely">Extremely Likely</string>
+    <string name="thanks_message">Thanks for taking the survey</string>
```
Comment (hoangnguyen92dn) [inline android/src/main/res/values/strings.xml:17]: `.` :trollface: 

image

Comment (suho) [reply android/src/main/res/values/strings.xml]: Fixed in https://github.com/suho/kmm-ic/pull/124/commits/3b7b880311e92f9501eb7218bc517f9bcd6f6112

File: android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyThanksScreen.kt [hunk 2]
```diff
@@ -1,16 +1,70 @@
...
+        if (progress == 1.0f) {
+            delay(500L)
+            onFinishedAnimation()
+        }
+    }
+
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.Black)
```
Comment (luongvo) [inline android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyThanksScreen.kt:40]: the screen background color is not purely black

image

File: android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyThanksScreen.kt [hunk 3]
```diff
@@ -1,16 +1,70 @@
...
+    ) {
+        Column(
+            verticalArrangement = Arrangement.Center,
+            modifier = Modifier
+                .statusBarsPadding()
```
Comment (luongvo) [inline android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyThanksScreen.kt:45]: Do we need this? how about the bottom navigation bar as well?

**Review by luongvo — COMMENTED** [review]
rest lgtm

File: android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyThanksScreen.kt (hunk 3 above)
Comment (suho) [reply android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyThanksScreen.kt]: Fixed in https://github.com/suho/kmm-ic/pull/124/commits/55e36bcdb51d375600e5f469cad73821516cecb6

File: android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyThanksScreen.kt (hunk 2 above)
Comment (suho) [reply android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyThanksScreen.kt]: Fixed in https://github.com/suho/kmm-ic/pull/124/commits/55e36bcdb51d375600e5f469cad73821516cecb6

**Review by luongvo — APPROVED** [review]

**Pull request merged**